apps/src/templates/accessible-dialogue.module.scss

```diff
@@ -24,3 +24,47 @@
   padding: 1rem;
   overflow: auto;
 }
+
```

Comment:
We currently have [12 use cases](https://github.com/search?q=repo%3Acode-dot-org%2Fcode-dot-org+AccessibleDialog&type=code) of the `AccessibleDialog` component. Most of those render an X close icon and  then [duplicate the same css](https://github.com/search?q=repo%3Acode-dot-org%2Fcode-dot-org%20.xCloseButton%20&type=code) in individual scss modules. Let's bake the X icon into the component. 
I started down that path by co-locating the X icon styling with the rest of the styles for the accessible dialog/

Reply:
@levadadenys Is a Dialog/Modal like this in the works for the Design System Component Library? I think it would be great to standardize styling and de-duplicate code across these use cases, but don't want to do throw away work if there is an new and improved component already planned. Thanks!

Reply:
@code-dot-org/teacher-tools looks like ya'll own a handful of the other use cases for `AccessibleDialog` with an X close icon styled the same way (e.g. https://github.com/code-dot-org/code-dot-org/pull/56874) . Any initial objections if I try to consolidate?